### DISCUSSION TITLE
請問有沒有繁體字本位的碼表？

### DISCUSSION THREAD

**Discussion by Namarimizu** [siuze/ShanRenMaLTS#45] [想法]
如題，之前其他山人方案的繁體本位版直接挪用效果並不好，小問題很多。
需求大概是完全去除簡化字，或者簡化字權重更低，x 這種簡碼也安排給「爲」而不是「对」這樣
沒有的話打算自己動手慢慢弄一份了……

**Comment by siuze** [comment]
本方案不能满足您的需求。   
可以尝试自行修改项目仓库的单字表文件，删除简化字或者调整简化字权重，然后再自己运行python脚本来生成码表。  
需要注意的是本方案是简体词库，如果直接删掉简体单字，会造成生成词库编码时出问题。
